src/main/java/no/unit/nva/cristin/projects/Constants.java

```diff
@@ -0,0 +1,17 @@
+package no.unit.nva.cristin.projects;
+
+import nva.commons.core.Environment;
+
+public class Constants {
+
+    private static final Environment ENVIRONMENT = new Environment();
+    private static final String CRISTIN_API_HOST_ENV = "CRISTIN_API_HOST";
+    public static final String PROJECT_CONTEXT_URL = "https://example.org/search-api-context.json";
```

Comment:
Is this a test value?

Reply:
No its an "actual" value according to datamodel, but I dont know how dynamic it should be. @brinxmat What is your opinion?

Reply:
I will update the relevant PR so that <https://bibsysdev.github.io/src/project-context.json> works